.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: php
-
-sudo: false
-
+group: edge
+sudo: required
+dist: trusty
```

Comment:
this should be done only for the HHVM build rather than for PHP builds too, as the docker-based infra is faster to boot. See how it is done in Symfony